.storybook/deprecated/cyclebutton/cyclebutton.stories.js

```diff
@@ -1,7 +1,7 @@
 import { default as ActionButtonStories } from "@spectrum-css/actionbutton/stories/actionbutton.stories.js";
 import { Template as ActionButton } from "@spectrum-css/actionbutton/stories/template.js";
 import { default as IconStories } from "@spectrum-css/icon/stories/icon.stories.js";
-import { useArgs } from "@storybook/client-api";
+import { useArgs } from "@storybook/preview-api";
```

Comment:
`@storybook/client-api` has been deprecated for Storybook 8, and its contents are now located within `@storybook/preview-api. You'll see this update in several other `template.js` files in this diff.